tools/dm/dm-upgrade.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Data Migration Upgrade
+summary: Lear how to upgrade a Data Migration version to a non-compatible version.
+category: tools
+---
+
+# Data Migration Upgrade
+
+This document introduces how to upgrade your Data Migration (DM) version to an incompatible version.
+
+Assuming that V-A, V-B, V-C are three DM versions in chronological order and they are not compatible with each other, you need to upgrade V-A to V-C. Upgrade-A-B means upgrading V-A to V-B and Upgrade-B-C means upgrading V-B to V-C.
+
+- If Upgrade-A-B overlaps Upgrade-B-C (e.g. different changes of a same configuration item), it is recommended to perform Upgrade-A-B to V-B and then perform Upgrade-B-C to V-C.
+- If Upgrade-A-B doe not overlap Upgrade-B-C, you can merge Upgrade-A-B and Upgrade-B-C as Upgrade-A-C to upgrade DM from V-A to V-C.
+
+> **Note:**
+>
+> - Generally, DM version upgrade means upgrading DM from the previous version with an upgrade procedure to the current version.
+> - Generally, all the following upgrade examples assume that you have downloaded the corresponding DM version and DM-Ansible version, and the DM binary exists in the corresponding directory of DM-Ansible. (For how to download the DM binary, see [Upgrade the component version](/tools/dm/cluster-operations.md#upgrade-the-component-version)).
+> - Generally, all the following upgrade examples assume that all the data replication tasks have been stopped before the upgrade and all the replication tasks are restarted manually after DM upgrade is finished.
```

Comment:
```suggestion
> - Unless otherwise stated, all the following upgrade examples assume that all the data replication tasks have been stopped before the upgrade and all the replication tasks are restarted manually after DM upgrade is finished.
```